source/braille.py

```diff
@@ -920,13 +928,13 @@ def update(self):
 		super(TextInfoRegion, self).update()
 
 		if self._selectionStart is not None:
-			# Mark the selection with dots 7 and 8.
+			# Mark the selection.
 			if self._selectionEnd >= len(self.rawText):
 				brailleSelEnd = len(self.brailleCells)
 			else:
 				brailleSelEnd = self.rawToBraillePos[self._selectionEnd]
 			for pos in xrange(self.rawToBraillePos[self._selectionStart], brailleSelEnd):
-				self.brailleCells[pos] |= DOT7 | DOT8
+				self.brailleCells[pos] |= SELECTION_SHAPE

```

Comment:
The documentation for Show cursor says it affects the selection indicator, but it doesn't. I assume the documentation reflects your intent and the code does not? :)